packages/example-next/connectors/walletConnect.ts

```diff
@@ -1,13 +1,14 @@
 import { initializeConnector } from '@web3-react/core'
 import { WalletConnect } from '@web3-react/walletconnect'
-import { URLS } from '../chains'
+import { MAINNET_CHAINS } from '../chains'
 
 export const [walletConnect, hooks] = initializeConnector<WalletConnect>(
   (actions) =>
     new WalletConnect({
       actions,
       options: {
-        rpc: URLS,
+        projectId: 'a6cc11517a10f6f12953fd67b1eb67e7',
```

Comment:
maybe make this a const outside the function or an env variable?

Reply:
Yeah, that's the plan. I will actually move it to `.env`